docs/DeveloperGuide.md

```diff
@@ -225,6 +225,79 @@ and South-East Asian universities. This command hence helps the users to keep tr
 #### Sequence Diagram on PlantUML
 ![Add Courses Sequence Diagram](../uml-images/AddCoursesCommand.png)
 
+### 6. List Commands Command
+
+#### Overview:
+The `ListCommandsCommand` provides users with a comprehensive list of all available commands in the CLI. This is particularly useful for new users or those unfamiliar with specific command formats.
+
+#### How the feature is implemented:
+* The `ListCommandsCommand` class extends the `Command` superclass and overrides the `execute` method.
```

Comment:
Since ListCommandsCommand extends CheckInformationCommand, I am not too sure if can just write extend Command class. Otherwise, everything else LGTM!

Reply:
Nice eye!